libs/dal/src/repositories/change/change.schema.ts

```diff
@@ -44,6 +42,16 @@ changeSchema.virtual('user', {
   justOne: true,
 });
 
+changeSchema.index({
+  _environmentId: 1,
+  _parentId: 1,
+  _entityId: 1,
+  createdAt: 1,
```

Comment:
Are we doing sorts with "-createdAt" ? depending on the order you might need to switch to "-1"

Reply:
We do, that's why I added it here so we don't do blocking sort.
And yes we do by 1